src/twitter_media_augment/index.js

```diff
@@ -88,9 +89,7 @@ module.exports = function (context, message) {
 
         return twit.get('search/tweets', params)
             .then(historyData => {
-                message.tweethistory_ids
-                    = historyData.statuses.map(statusItem => statusItem.id_str);
-
+                message.tweetHistoryData.statuses = historyData.statuses;
```

Comment:
Reviewers: The other property on historyData is searchMetadata (query, time to execute...)